dapps/evancore.vue.libs/src/components/profile/profile-preview/profile-preview.ts

```diff
@@ -219,4 +221,12 @@ export default class ProfilePreviewComponent extends mixins(EvanComponent) {
       FileHandler.fileToContainerFile(file)
     ));
   }
+
+  /**
+   * Overwrites the reset value, when the user cancels the edit mode.
+   */
+  setOriginalUserInfo() {
```

Comment:
I'd rename this. Functions starting with "set" will be expected to be setters. This is more of a "resetter" or "restorer".